### ISSUE TITLE
I want open ios photo api

### ISSUE THREAD

**Issue by kkgg0521** [emericg/QmlAppTemplate#5]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by emericg** [comment]
I'm sorry this is a very complicated problem, and not in the scope of this template.
I suggest you start with https://doc.qt.io/qt-6/qml-qtquick-dialogs-filedialog.html

**Comment by kkgg0521** [comment]
QML doesn't seem to be able to open iOS albums very well. Should I try using OBJC



发自我的iPhone


------------------ Original ------------------
From: Emeric ***@***.***&gt;
Date: Thu,Feb 6,2025 11:19 PM
To: emericg/QmlAppTemplate ***@***.***&gt;
Cc: Lvmoney ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [emericg/QmlAppTemplate] I want open ios photo api (Issue #5)





 
I'm sorry this is a very complicated problem, and not in the scope of this template.
 I suggest you start with https://doc.qt.io/qt-6/qml-qtquick-dialogs-filedialog.html

**Comment by emericg** [comment]
Yes, it's a real challenge to do that in Qt/QML, the solution probably involves a fair amount of native code. And to my knowledge, this has never been fully solved.

**Comment by kkgg0521** [comment]
Thank you very much, I will try it

**Comment by kkgg0521** [comment]
I need to get the picture content in the photo album in iOS. If you have a good solution, I hope we can discuss it together. Thanks

**Comment by emericg** [comment]
You can also look at this code https://github.com/ekke/ekkesSHAREexample. This is based on an old Qt version, is incomplete feature wise, but it has a lot of interesting things in it as well as technical info on the underlying OS code. It will take a significant amount of work to make it run.

And I would still advise you to start with the qml file dialog.
